src/components/Config/Config.tsx

```diff
@@ -30,11 +29,11 @@ function Config() {
   } = useMemo(
     () => ({
       0: {
```

Comment:
Now that you have the `ConfirmationTime` enum, the point is to use it here too. Again, that way you only use `1d` `3d` etc instead of decoding `1d` into 1 and vice versa

Reply:
And to use those enums here the syntax is `[ConfirmationTime.immediate]: { ... }` just FYI